airbyte-integrations/connectors/source-railz/source_railz/manifest.yaml

```diff
@@ -0,0 +1,362 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer: ['data']
+  requester:
+    type: HttpRequester
+    url_base: "https://api.railz.ai/"
+    http_method: "GET"
+    authenticator:
+      class_name: source_railz.components.ShortLivedTokenAuthenticator
+      url: "https://auth.railz.ai/getAccess"
+      client_id: "{{ config.client_id }}"
+      secret_key: "{{ config.secret_key }}"
+      lifetime: "PT3600S"
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: DefaultPaginator
+      url_base: "*ref(definitions.requester.url_base)"
+      page_size_option:
+        inject_into: request_parameter
+        field_name: "limit"
+      pagination_strategy:
+        type: OffsetIncrement
+        page_size: 100
+      page_token_option:
+        inject_into: request_parameter
+        field_name: "offset"
+    requester:
+      $ref: "*ref(definitions.requester)"
+
+# Full Refresh Base
+  base_stream:
+    type: DeclarativeStream
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  businesses_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "businesses"
+      path: "/businesses"
+    primary_key: "businessName"
+  connections_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "connections"
+      path: "/connections"
+    primary_key: "connectionId"
+
+# Internal stream for slicing
+  connections_slice_stream:
+    $ref: "*ref(definitions.base_stream)"
+    retriever:
+      $ref: "*ref(definitions.base_stream.retriever)"
+      record_selector:
+        $ref: "*ref(definitions.selector)"
+        record_filter:
+          condition: "{{ record.status in ['active', 'disconnected', 'expired'] and record.serviceName in options.serviceNames }}"
```

Comment:
could you leave a comment in the YAML explaining why we filter for active/disconnected/expired and serviceNames?